contracts/BSCValidatorSet.sol

```diff
@@ -73,6 +73,26 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     Validator[] validatorSet;
   }
 
+  // BEP-99 Temporary Maintenance
+  uint256 public constant EXIT_MAINTENANCE_REWARD_RATIO_SCALE = 100;
+  uint256 public constant EXIT_MAINTENANCE_REWARD_RATIO = 20;
+  uint256 public constant SAFE_MIN_GAS = 2300;
+
+  uint256 public MaxNumOfMaintaining;
+  uint256 public MaxMaintainingTime;
+
+  struct MaintainInfo {
+    // TODO optimize struct
+    bool isMaintaining;
+    uint256 maintainStartAt;
+    uint256 maintainingIndex;   // the index of the element in `maintainingValidatorSet`.
+    uint256 exitMaintenanceReward;
```

Comment:
change to :
```
  struct MaintainInfo {
    uint256 startAt; // Must be larger than 0
    uint256 index;   // the index of the element in `maintainingValidatorSet`.
    uint256 exitReward;
}
```
if `startAt`==0, it means not in maintain.

Reply:
```
`startAt`==0 means has not maintained  during this 24 hours period of time.
if `isMaintaining` removed,  there's no way to know if it's maintaining

1. before enterMaintenance:  `startAt`==0,   `isMaintaining` == false
2. Maintaining: `startAt` != 0,   `isMaintaining` == true
3. after exitMaintenance:   `startAt` != 0,   `isMaintaining` == false



```